### ISSUE TITLE
After merging audio and video, audio and video stay around

### ISSUE THREAD

**Issue by Kyle-Mickan** [p-ranav/saveddit#16]
When download a file that separates audio and video into 2 files after saveddit merges them into 1 file the individual audio and video files stay around. Is this intended functionality? or Can there be an option to only keep the merged file when downloading?

**Comment by p-ranav** [comment]
Yeah `saveddit` right now doesn't delete the individual audio and video files after a successful merge - I figured some might like the video component without audio so I decided against deleting it at the time of development.

I think it can probably be changed to always delete the individual components if the merge is successful.

**Comment by NickolaiBeloguzov** [comment]
I think it needs to be an option. Like, by default saveddit can keep these files after merging, but if user provides a flag _(e.g. **--clean** or something like this)_ these files will be deleted. This behaviour could be reversed _(deleting by default, keeping if user requests)_
